From 1c7de9adfb10f8d8f92c211d0d97b133d43799fb Mon Sep 17 00:00:00 2001
From: Paul Naoumenko <paulx.naoumenko@intel.com>
Date: Wed, 12 Dec 2012 12:43:44 +0100
Subject: [PORT FROM MAIN] Revert "wifi: [JB MR1 Bring Up] REVERT ME: temporarily relocate nvs.bin file"

BZ: 70883

This reverts commit f42a9b35f07018170a7f2220fa92ca475dd525ac.

/factory partition has been restored

Change-Id: I9da1a17fe66355bf65774074da366dc14375fd3f
Orig-Change-Id: Ic1bcaea0a7820765d23d6c78908db7133ff33d00
Signed-off-by: Paul Naoumenko <paulx.naoumenko@intel.com>
Reviewed-on: http://android.intel.com:8080/81716
Reviewed-by: Bachot, Jean-michel <jean-michel.bachot@intel.com>
Tested-by: Zurmely, PierreX <pierrex.zurmely@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 wlan_prov/Android.mk          |    2 +-
 wlan_prov/wlan_provisioning.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/wlan_prov/Android.mk b/wlan_prov/Android.mk
index c123fa6..162a584 100644
--- a/wlan_prov/Android.mk
+++ b/wlan_prov/Android.mk
@@ -36,7 +36,7 @@ $(SYMLINKS): $(LOCAL_INSTALLED_MODULE) $(LOCAL_PATH)/Android.mk
 	@echo "Symlink: $@"
 	@mkdir -p $(dir $@)
 	@rm -rf $@
-	$(hide) ln -sf $(addprefix /data/misc/wifi/,$(NVS_FILE)) $@
+	$(hide) ln -sf $(addprefix /factory/wifi/,$(NVS_FILE)) $@
 
 ALL_DEFAULT_INSTALLED_MODULES += $(SYMLINKS)
 ALL_MODULES.$(LOCAL_MODULE).INSTALLED := \
diff --git a/wlan_prov/wlan_provisioning.c b/wlan_prov/wlan_provisioning.c
index 1df7dfd..c5a153d 100644
--- a/wlan_prov/wlan_provisioning.c
+++ b/wlan_prov/wlan_provisioning.c
@@ -37,7 +37,7 @@
 const unsigned char NullMacAddr[MAC_ADDRESS_LEN] = { 0, 0, 0, 0, 0, 0 };
 
 #define WIFI_PATH "/data/misc/wifi"
-const char NVS_file_name[] = "/data/misc/wifi/wl1271-nvs.bin";
+const char NVS_file_name[] = "/factory/wifi/wl1271-nvs.bin";
 const char Default_NVS_file_name[] = "/system/etc/wifi/wl1271-nvs.bin";
 const char WLAN_SDIO_BUS_PATH[] = "/sys/bus/sdio/drivers/wl1271_sdio/";
 #define WLAN_DRV_SDIO_NAME "wl1271_sdio"
-- 
1.7.4.1

